### PR TITLE
Update aws and DO Kubernetes deploy docs

### PR DESCRIPTION
**Pull request by dasiklasik**

### CODE REVIEW

File: deploy/aws/deploy/app/api/templates/deployment.yaml [hunk 1]
```diff
@@ -19,6 +19,9 @@ spec:
       - name: {{ .Values.service }}
         image: {{ .Values.awsAccountId }}.dkr.ecr.{{ .Values.awsRegion }}.amazonaws.com/{{ .Values.service }}:{{ $.Values.imagesVersion }}
         imagePullPolicy: Always
+        envFrom:
+          - secretRef:
+              name: secret_name
```
Comment (fruneen) [inline deploy/aws/deploy/app/api/templates/deployment.yaml:24]: hmm, let's create a secret ref name dynamically:

Pass in helm upgrade command (`deploy/aws/deploy/script/src/index.js` line 56) current environment (staging/production) 

And update secret ref name using variables:
```suggestion
              name: api-{{ .Values.environment }}-secret
```

also, we need to configure ConfigMaps using (differences here: https://k21academy.com/docker-kubernetes/configmaps-secrets/#:~:text=What%20is%20the%20difference%20between,API%20keys%2C%20or%20TLS%20certificates.)

File: deploy/aws/deploy/bin/deploy-secrets.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+read -p "What stage? (staging, production) " stage
+
+echo "Set up necessary environment variables"
+read -p "Write MONGO_URI "  mongo_uri
+
+read -p "Write MONGO_DB_NAME " mongo_db_name
+
+read -p "Write API_URL " api_url
+
+read -p "Write WEB_URL " web_url
```
Comment (fruneen) [inline deploy/aws/deploy/bin/deploy-secrets.sh:14]: let's remove it, because we need to use this script for updating env variables in existing application

**Review by fruneen — CHANGES_REQUESTED** [review]

File: deploy/aws/deploy/bin/deploy-secrets.sh
Comment (fruneen) [inline deploy/aws/deploy/bin/deploy-secrets.sh:1]: perfect!

Maybe we will add a log, that secret was successfully created?

File: docs/deployment/kubernetes/aws.mdx [hunk 3]
```diff
@@ -398,6 +384,72 @@ eksctl create iamidentitymapping --cluster my-app --arn arn:aws:iam::40216744126
 
 In the **--arn** parameter you need to specify your user ARN, which you can find in the IAM dashboard.
 
+## Environment variables
```
Comment (fruneen) [inline docs/deployment/kubernetes/aws.mdx:387]: You can use Reusable snippets for not to duplicate info about k8s envs: https://mintlify.com/docs/content/reusable-snippets

**Review by fruneen — CHANGES_REQUESTED** [review]
Amazing 🤩

File: deploy/aws/deploy/bin/deploy-secrets.sh
Comment (dasiklasik) [reply deploy/aws/deploy/bin/deploy-secrets.sh:1]: Commands for creating and updating secrets and configmap return logs about the result of operation (examples in screenshots)
image
image

**Pull request merged**